sdk/communication/AzureCommunicationChat/Source/Generated/Models/ChatError.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 // swiftlint:disable cyclomatic_complexity
 
 /// The Communication Services error.
-public final class CommunicationError: Codable, Swift.Error {
+public final class ChatError: Codable, Swift.Error {
```

Comment:
Is this suppose to conform to `AzureError`?

Reply:
Just to clarify because ChatError is generated, to inherit from AzureError is this change on autorest side or something we need to reference in the swagger?

Reply:
ChatError has a different purpose, it is not intended to be used as an exception. You can look at the other languages on how it is used.  
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Chat/src/Generated/Models/ChatError.cs